### PR TITLE
Add features to linux gradle, update readme to match

### PR DESCRIPTION
**Pull request by S-B99**

### CODE REVIEW

**Comment by zeroeightysix** [comment]
Can you put the new tasks into a group `kami`?
```gradle
task myTask() {
    group = "kami"
    ...
}
 ```
And please remove `testprint`

**Comment by S-B99** [comment]
Forgot `testprint`, and just a sec on groups thanks

File: build.gradle [hunk 1]
```diff
@@ -1,3 +1,31 @@
...
+task mkdir {
+    group = "kami"
+    doLast {
+        mkdir "../../.minecraft/mods/1.12.2"
+    }
+}
+
+// delete old versions
+
+task rmold(type: Delete) {
```
Comment (zeroeightysix) [inline build.gradle]: Can this be camel case? `rmOld`

File: build.gradle [hunk 2]
```diff
@@ -1,3 +1,31 @@
...
+        include "KAMI*.jar"
+    }
+}
+
+// copy file
+
+task copy(type: Copy) {
+    group = "kami"
+    from file("build/libs/KAMI-b9-release.jar")
+    into file("../../.minecraft/mods/1.12.2/")
```
Comment (zeroeightysix) [inline build.gradle]: We can't just assume `../../.minecraft` is the correct path to our minecraft directory.
Use `System.getProperty("user.home") + "/.minecraft"` instead

Comment (S-B99) [reply build.gradle]: I'm not sure how reviews work on github, do I edit it myself?

Comment (zeroeightysix) [reply build.gradle]: Just commit and mark the conversation as resolved.

File: README.md [hunk 3]
```diff
@@ -74,24 +76,33 @@ If you do not wish to run from an IDE, use `gradlew.bat runClient` to run KAMI.
...
+You can build by running these commands (without these <>) in a terminal with the current directory being KAMI. (EG. `cd C:\Users\Username\Downloads\KAMI`)
 ```
-gradlew.bat build
-cd build/libs
+gradlew.bat <args>
 ```
-To copy the release  jar in `build/libs` you can run `autocopy.bat` or, if you prefer doing it manually, find a file `KAMI-<minecraftVersion>-<kamiVersion>-release.jar` which you can copy to the `mods` folder of a minecraft instance that has forge installed.
+Possible arguments on Windows are `build`.
+
+To copy on windows run `autocopy.bat`
```
Comment (zeroeightysix) [inline README.md]: If there's only one argument possible, just have the instructions be:
```
gradlew.bat build
```

Also, we're under the `Windows` section: no need to mention `To copy on windows`

File: README.md [hunk 4]
```diff
@@ -74,24 +76,33 @@ If you do not wish to run from an IDE, use `gradlew.bat runClient` to run KAMI.
...
-cd KAMI
-chmod +x auto*
-./autobuild
-mkdir ~/.minecraft/mods
-./autocopy
+./gradlew <args>
 ```
+Possible arguments are `build`, `mkdir`, `rmOld` and `copy`, in that order. 
+
+Build is required, mkdir makes the mods/1.12.2 directory, rmOld removes old versions of KAMI in that directory, and copy copies the build release to the mods/1.12.2 directory.
```
Comment (zeroeightysix) [inline README.md]: Reformat the names to be in `code blocks`?

**Comment by leijurv** [comment]
hey cis scum, not everyone is `~/.minecraft/`

on mac osx it's `~/Library/Application Support/minecraft/`

**Comment by leijurv** [comment]
hey not everyone is `~/.minecraft/`

on mac osx it's `~/Library/Application Support/minecraft/`

**Comment by leijurv** [comment]
hey not everyone is `~/.minecraft/`

on mac osx it's `~/Library/Application Support/minecraft/`

**Comment by S-B99** [comment]
> hey cis scum

how nice of you. and where did I say I was supporting osx? That's coming later, separately

**Comment by blockparole** [comment]
An easy fix for compatibility would be checking `$OSTYPE`, also support for MultiMC would be awesome :3

**Comment by S-B99** [comment]
I have compatibility down, MultiMC with would have to be another check but it's possible. I'm not sure MultiMC sets env variables, especially not on stuff other then Linux

File: README.md (hunk 4 above)
Comment (S-B99) [reply README.md]: Updated

File: README.md (hunk 3 above)
Comment (S-B99) [reply README.md]: Updated

**Comment by S-B99** [comment]
I'm not sure how to check, does this pull include my gradlew and the windows changes? [\(Follow this to see all differences\)](https://stackoverflow.com/questions/1968512/getting-the-difference-between-two-repositories). 
If it does, you can force the pull and replace my readme with yours. The only changes I made that you can't keep is adding you to thank yous, and adding my issues button in the readme.

**Comment by S-B99** [comment]
The command is the other way around. Open this with the diff syntax in sublime. + is stuff I added
[diff.txt](https://github.com/zeroeightysix/KAMI/files/3693662/diff.txt)

**Comment by S-B99** [comment]
On line 38 in [commit](https://github.com/S-B99/KAMI/commit/1dfd30444b280188d1a37353b320b11f108b4816) posVec is still using Vector, is this fine @zeroeightysix? 
I'm not sure who made the change to stop using it, I'm just confirming because I'm readding all of the changes between my repo and yours

**Comment by S-B99** [comment]
I will keep the suggestions as reference, though due to this drifting too far from 086's version I will manually reimplement everything and continue the project. Thanks for all of your hard work everybody.

**Pull request closed without merging**